lib/Helper/File.ts

```diff
@@ -54,7 +60,7 @@ export function matchesContent(
     return false;
   }
   return matches.reduce((prev: boolean, match: string) => {
-    return (
+    return !!(
```

Comment:
Should we rename the function if now we are using the double not?

Reply:
The function was and is still returning boolean. The reason for the `!!` is ts typings, without the `ts` the boolean return value is not recognize. I would not rename is since it's still perform exactly the same function.